delivery-slot-management/README.md

```diff
@@ -0,0 +1,140 @@
+
+- [Using the Installer Recipes](https://help.sap.com/docs/SAP_COMMERCE_CLOUD_PUBLIC_CLOUD/8be4e0379b294fbabf36d26e7ca4169d/cb3ff964e4784073a70f06165efbac8a.html?locale=en-US&version=v2205) to install using the cx recipe
+- [Installing SAP Commerce Cloud Manually](https://help.sap.com/docs/SAP_COMMERCE_CLOUD_PUBLIC_CLOUD/8be4e0379b294fbabf36d26e7ca4169d/8bf5cfea86691014a00e9705a3c84074.html?locale=en-US&version=v2205) to install manually if you know the required extensions to set up the electronics or apparel-uk accelerator
+
+### Install the Easy Extension Framework
+1. Download [Easy Extension Framework](https://github.com/SAP/easy-extension-framework)
+2. Add Easy extensions to your `localextensions.xml` file:
+```
+<extension dir="easycore" />
+<extension dir="easytype" />    
+<extension dir="easybackoffice" />    
+<extension dir="easyrest" />
+```
+3. Run `ant clean all`
+4. Run `ant initialize -Dtenant=master`
+5. Start SAP Commerce Cloud server with `<HYBRIS_HOME>/bin/platform/hybrisserver.sh`
+
+### Configure local Easy repository
+Next you'll have to configure the Git repository where the tutorial extension is stored. If you want to play around locally with the extension we suggest you to create a local easy repository that will simplify and speed up your development.
+Follow these steps:
+- Clone locally the [Easy Extension Sample](https://github.tools.sap/cx-boosters/easy-extension-samples/tree/easy-0.2) repository (branch easy-0.2)
+- Login into the backoffice and find the Easy node on the Navigation Tree, open Easy Core, Repositories and click on the + button to create a new "local repository"
+![img.png](./images/img.png)
+- populate the wizard giving the repo a code, name and the path of your local File System where you've cloned the remote easy-extension-samples repo
+![img_1.png](./images/img_1.png)
+
+### Install Delivery Slot Management Easy extension
+Easy extensions have to be installed through the Administration Console:
+- Connect to the hac and log in
+- If you've correctly created your Easy local repository, you should see under the Easy tab, the repository with the list of easy extensions retrieved:
+![img_2.png](./images/img_2.png)
+- Click the "Update" button to be sure you've the latest version of your local repository: the update button will refresh the copy of the repositories that is stored in the _data_ dir of Commerce
+- Click on the "+" button next to the "Delivery Slot Management" extension and wait the installation procedure is completed: if there are no errors you should see a successful blue message banner in the hac, if something went wrong during installation, a red message banner will pop up in the hac. In this case you'll have to check the server log to see what is the problem
+- We suggest you to repeat the last step also for the "Easy Api" extension that will give you access to a Swagger endpoint so that you can easily test the endpoints of the tutorial extension 
+
+## How to test
```

Comment:
Link this to: https://sap.github.io/easy-extension-framework/configuring-an-easy-repository-in-backoffice/